src/mod/applications/mod_conference/mod_conference.c

```diff
@@ -3757,6 +3757,7 @@ conference_obj_t *conference_new(char *name, conference_xml_cfg_t cfg, switch_co
 	switch_thread_rwlock_create(&conference->rwlock, conference->pool);
 	switch_mutex_init(&conference->member_mutex, SWITCH_MUTEX_NESTED, conference->pool);
 	switch_mutex_init(&conference->canvas_mutex, SWITCH_MUTEX_NESTED, conference->pool);
+	switch_mutex_init(&conference->relate_mutex, SWITCH_MUTEX_NESTED, conference->pool);
```

Comment:
Let's add corresponding `switch_mutex_destroy` ?

Reply:
We can explicitly destroy the relate_mutex but since they are created using conference pool, mutex should be release upon deleting mem pool. The author didn't explicitly destroy those other mutex and I have followed his convention.

Reply:
`switch_mutex_destroy` should be called in module unload function on all those mutexes but author skipped that with pragmatic approach, I do not insist.